mono/metadata/image.c

```diff
@@ -2605,4 +2605,4 @@ mono_find_mempool_owner (void *ptr)
 	return owner;
 }
 
-#endif
\ No newline at end of file
+#endif
```

Comment:
It's fine to add a newline at the end of files that don't have it so that we can avoid formatting noise from editors that add it automatically (which is most of them). Please split that into a separate commit, though.